packages/bsky/src/auth-verifier.ts

```diff
@@ -236,16 +251,19 @@ export class AuthVerifier {
   }
 
   parseCreds(
-    creds: StandardOutput | RoleOutput | AdminServiceOutput | NullOutput,
+    creds: StandardOutput | RoleOutput | ModServiceOutput | NullOutput,
   ) {
     const viewer =
       creds.credentials.type === 'standard' ? creds.credentials.iss : null
     const canViewTakedowns =
       (creds.credentials.type === 'role' && creds.credentials.admin) ||
-      creds.credentials.type === 'admin_service'
+      creds.credentials.type === 'mod_service' ||
+      (creds.credentials.type === 'standard' &&
+        this.isModService(creds.credentials.iss))
```

Comment:
Yup that does sound right - maybe it could be communicated better.

But the latter comes into play on some of the view routes where we still want to show taken down content - `getProfile` for instance

Reply:
We could use a separate authVerifier there like `optionalStandardOrModService` but I thought it was a bit clearer to just use `optionalStandard` and then do a follow on check to see if it came from a trusted did